sendgrid/model/email.py

```diff
@@ -0,0 +1,106 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Roman Zoller
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp import models, fields, api, exceptions, _
+from openerp.tools.config import config
+
+import logging
+import sendgrid
+
+
+SUBSTITUTION_PREFIX = '{'
```

Comment:
This means someone has to change the code if he wants to change the prefix and postfix. I think it would be better ton include this in the system parameters.